src/pudl_zenodo_storage/zs/metadata.py

```diff
@@ -20,11 +20,26 @@
 """
 
 
+def _parse_contributor_metadata(pudl_contributors) -> List[dict]:
+    """Reformat PUDL contributor metadata to fit Zenodo requirements."""
+    zenodo_cont_list = []
+
+    for contributor in pudl_contributors:
+        pudl_cont_dict = contributor.dict()
+        zenodo_cont_dict = {
+            "name": pudl_cont_dict["title"],
+            "affiliation": pudl_cont_dict["organization"],
+        }
+        zenodo_cont_list.append(zenodo_cont_dict)
+
+    return zenodo_cont_list
+
+
 def _generate_metadata(data_source_id, data_source_uuid):
-    if data_source_id == "eipinfrastructure":
-        data_source = DataSource(**pzs.frictionless.eipinfrastructure.eipinfrastructure)
-    else:
-        data_source = DataSource.from_id(data_source_id)
+    # if data_source_id == "eipinfrastructure":
+    #    data_source = DataSource(**pzs.frictionless.eipinfrastructure.eipinfrastructure)
+    # else:
```

Comment:
Commented out?

Reply:
I figured we don't use eip data anymore

Reply:
Ah okay. If we don't use it anymore we should just delete it -- the old versions are preserved in the repository history.